code/modules/events/spider_terror.dm

```diff
@@ -0,0 +1,30 @@
+/var/global/sent_terror_spiders_to_station = 0
+
+/datum/event/spider_terror
+	announceWhen	= 400
+	var/spawncount = 1
+
+/datum/event/spider_terror/setup()
+	announceWhen = rand(announceWhen, announceWhen + 50)
+	spawncount = 1 //round(num_players() * 0.8)
+	sent_terror_spiders_to_station = 1
+
+/datum/event/spider_infestation/announce()
+	command_announcement.Announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", new_sound = 'sound/AI/aliens.ogg')
+
+/datum/event/spider_terror/start()
+
+	var/list/vents = list()
+	for(var/obj/machinery/atmospherics/unary/vent_pump/temp_vent in world)
```

Comment:
Even though https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/events/spider_infestation.dm does it the exact same way?

I'm not actually sure how to rewrite this one without the "in world".